pkg/util/http.go

```diff
@@ -22,7 +22,7 @@ func MakeBackupURL(host, version string) *url.URL {
 	u := &url.URL{
 		Scheme: "http",
 		Host:   host,
-		Path:   "/backup",
+		Path:   "/v1/backup",
```

Comment:
there is the import cycle. i think you originally move this out to avoid import cycles. also /backup, /backupnow are still not using as const. it makes no sense to have backup http client related stuff in util actually.

we need a large code refactor to sort this out. see https://github.com/coreos/etcd-operator/issues/682